src/util/file_download.ts

```diff
@@ -8,3 +10,6 @@ export const fileDownload = (href: string, filename = ""): void => {
   a.dispatchEvent(new MouseEvent("click"));
   document.body.removeChild(a);
 };
+
+export const downloadFileSupportedViaApp = (hass: HomeAssistant): boolean =>
+  !!hass.auth.external?.config.downloadFileSupported;
```

Comment:
this will indicated android app doesn't support downloads, while it actually does?

Reply:
Yes you are right. I can move the isIOs check in here and rename it for ios only